src/plugins/add-cal/reports/a11y-1.json

```diff
@@ -0,0 +1,921 @@
+        "WCAG21Tech:failures/F24",
+        "WCAG21Tech:failures/F83"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.4.4 Resize text (Level AA)",
+      "earl:test": "WCAG20:resize-text",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G142",
+        "WCAG21Tech:general/G146",
+        "WCAG21Tech:general/G178",
+        "WCAG21Tech:general/G179",
+        "WCAG21Tech:failures/F69",
+        "WCAG21Tech:failures/F80",
+        "WCAG21Tech:failures/F94"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.4.5 Images of Text (Level AA)",
+      "earl:test": "WCAG20:image-of-text",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G140",
+        "WCAG21Tech:css/C22",
+        "WCAG21Tech:css/C30"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.4.10 Reflow (Level AA)",
+      "earl:test": "WCAG21:reflow",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:css/C32",
+        "WCAG21Tech:css/C31",
+        "WCAG21Tech:css/C33",
+        "WCAG21Tech:css/C38",
+        "WCAG21Tech:client-side-script/SCR34",
+        "WCAG21Tech:general/G206",
+        "WCAG21Tech:failures/F102"
+      ],
+      "advisory": ["WCAG21Tech:css/C34", "WCAG21Tech:css/C37"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.4.11 Non-text Contrast (Level AA)",
+      "earl:test": "WCAG21:non-text-contrast",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G195",
+        "WCAG21Tech:general/G174",
+
+        "WCAG21Tech:general/G207",
+        "WCAG21Tech:general/G209",
+
+        "WCAG21Tech:failures/F78"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.4.12 Text Spacing (Level AA)",
+      "earl:test": "WCAG21:text-spacing",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:css/C36",
+        "WCAG21Tech:css/C35",
+        "WCAG21Tech:failures/F104"
+      ],
+      "advisory": [
+        "WCAG21Tech:css/C8",
+        "WCAG21Tech:css/C21",
+        "WCAG21Tech:css/C28"
+      ],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.4.13 Content on Hover or Focus (Level AA)",
+      "earl:test": "WCAG21:content-on-hover-or-focus",
+      "outcome": "Pass",
```

Comment:
This is not applicable, there is no content that become visible then hidden on the "hover" or "focus" event.

Regarding the show/hide of the details/summary element this happen on the "click" event.

Reply:
Changes addressed in recent commit